pkg/operators/oci-handler/oci.go

```diff
@@ -309,6 +309,14 @@ func (o *OciHandlerInstance) init(gadgetCtx operators.GadgetContext) error {
 	}
 	r.Close()
 
+	// add extra info if requested
+	if gadgetCtx.ExtraInfo() {
```

Comment:
I suggest doing this after all the image operators have been instantiated and prepared. This operation should never fail, so let's first run those that could fail to avoid wasting time in case of an error during instantiation/preparation of operators.